server/src/main/java/org/elasticsearch/threadpool/ThreadPool.java

```diff
@@ -339,6 +337,7 @@ public ExecutorService executor(String name) {
      */
     @Override
     public ScheduledCancellable schedule(Runnable command, TimeValue delay, String executor) {
+        command = preserveContext(command);
```

Comment:
Should `preserveContext` be private now?

Reply:
I've removed the method from the Scheduler interface in 34a704d